src/App.tsx

```diff
@@ -1,95 +1,203 @@
-              <label htmlFor="toggle-completed">qwertyuio</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
+    <MyTodos.Provider value={myTodos as ContextPropsMyTodos}>
+      <FilteredTodos.Provider
+        value={myFilteredTodos as ContextPropsFilteredTodos}
+      >
+        <body className="todo-app">
+          <header className="todo-app__header">
+            {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
+            <div className="todo-app__title" onClick={handleCrossAllOut}>
+              Todos
             </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li className="editing">
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-editing" />
-              <label htmlFor="toggle-editing">zxcvbnm</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-
-          <li>
-            <div className="view">
-              <input type="checkbox" className="toggle" id="toggle-view2" />
-              <label htmlFor="toggle-view2">1234567890</label>
-              <button type="button" className="destroy" data-cy="deleteTodo" />
-            </div>
-            <input type="text" className="edit" />
-          </li>
-        </ul>
-      </section>
-
-      <footer className="footer">
-        <span className="todo-count" data-cy="todosCounter">
-          3 items left
-        </span>
-
-        <ul className="filters">
-          <li>
-            <a href="#/" className="selected">
-              All
-            </a>
-          </li>
-
-          <li>
-            <a href="#/active">Active</a>
-          </li>
-
-          <li>
-            <a href="#/completed">Completed</a>
-          </li>
-        </ul>
-
-        <button type="button" className="clear-completed">
-          Clear completed
-        </button>
-      </footer>
-    </div>
+            <form
+              className="todo-app__form"
+              onSubmit={handleFormSubmission}
+              id={formKey.toString()}
+            >
+              <input
+                className="todo-app__input"
+                name="todo-app__input"
+                value={input}
+                placeholder="Got anything to do?"
+                // eslint-disable-next-line jsx-a11y/no-autofocus
+                autoFocus
+                onChange={handleInput}
+              />
+            </form>
+          </header>
+          <main>
+            <TodoList />
+            {todos && todos.length > 0 && (
+              <article className="todo-list__footer">
+                <div className="todo-list__completion">
+                  {todos.filter((t: Todo) => !t.completed).length} left
```

Comment:
pls do not use such expressions in jsx, move them to separate consts
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**